war/src/main/less/base/layout-commons.less

```diff
@@ -69,10 +69,7 @@ body {
 }
 
 #side-panel {
-  padding: 1rem 0 2.5rem 0;
-  width: 300px;
   flex-shrink: 0;
-  border-right: 1px solid var(--panel-border-color);
```

Comment:
Unrelated?

Reply:
I wouldn't call it necessarily unrelated. That one removes the small gray delimiter that splits the sidepanel from the mainpanel. Appears to be a seamless transition integrating well with the new sidebar.

Reply:
If your PR changes what icons look like, a change of border color/existence is unrelated.

These PRs are difficult enough to review without adding changes to entirely different UI elements.